Documentation of static and dynamic variables.

hellohynes opened the pull request:
This adds in documentation for all static and dynamic variables within Stacks.

![Variable page preview](https://user-images.githubusercontent.com/1544281/41697521-2ec4e4d2-74e7-11e8-9a35-5acbf0179883.png)

Closes #12

balpha commented:
I've added a compilation & config section and moved the "dynamic tokens" over there. Hope it was okay to hijack your branch :)

hellohynes commented:
re: my updates —

1. Simplified the copy a bit.
2. Changed references to "static" to "constants" and "dynamic" to "configurations".
3. Re-combined all of these into one file. I don't think there's enough information yet for having two separate pages. Happy to discuss that, but having seen them as separate pages, I don't think there's a need for this.

hellohynes commented:
Since there's are solely documentation changes, I'm going to merge these into develop. We can split these out later if we think it's necessary.
